File: Bossinfo.Models.Hospital/his.ts

 //改NameSpace就好
 
    import { BossApiModel, Key } from '@boss/decorator/BossApiModel';
    import { I18nNamespace } from '@boss/decorator/i18n/i18nNs';
    @BossApiModel('api/his')
    @I18nNamespace('Model.EDI.his')
    export class his {
        
        @Key()// Column(Order = 0)// DatabaseGenerated(System.ComponentModel.DataAnnotations.Schema.DatabaseGeneratedOption.None)
        public ID: number;
        @Key()// Column(Order = 1)// StringLength(200)
        public Name: string;
        @Key()// Column(Order = 2)// StringLength(10)
        public Account: string;
        @Key()// Column(Order = 3)// StringLength(10)
        public Password: string;
        @Key()// Column(Order = 4)// DatabaseGenerated(System.ComponentModel.DataAnnotations.Schema.DatabaseGeneratedOption.None)
        public Status: number;
        // StringLength(500)
        public Remark: string;
        // Column(TypeName = "datetime2")
        public CreatedTime: Date;
        
        public CreatedBy: number;
        // Column(TypeName = "datetime2")
        public LastModifiedTime: Date;
        
        public LastModifiedBy: number;
    }
    
    /*
    "his": {
        
        "ID": "ID",
        "Name": "Name",
        "Account": "Account",
        "Password": "Password",
        "Status": "Status",
        "Remark": "Remark",
        "CreatedTime": "CreatedTime",
        "CreatedBy": "CreatedBy",
        "LastModifiedTime": "LastModifiedTime",
        "LastModifiedBy": "LastModifiedBy"
    }
    */
    

File: Bossinfo.Models.Hospital/his.cs
namespace Bossinfo.Models.Hospital
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("web_upl.his")]
    public partial class his
    {
        [Key]
        [Column(Order = 0)]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public long ID { get; set; }

        [Key]
        [Column(Order = 1)]
        [StringLength(200)]
        public string Name { get; set; }

        [Key]
        [Column(Order = 2)]
        [StringLength(10)]
        public string Account { get; set; }

        [Key]
        [Column(Order = 3)]
        [StringLength(10)]
        public string Password { get; set; }

        [Key]
        [Column(Order = 4)]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public int Status { get; set; }

        [StringLength(500)]
        public string Remark { get; set; }

        [Column(TypeName = "datetime2")]
        public DateTime? CreatedTime { get; set; }

        public long? CreatedBy { get; set; }

        [Column(TypeName = "datetime2")]
        public DateTime? LastModifiedTime { get; set; }

        public long? LastModifiedBy { get; set; }
    }
}